### PR TITLE
CDRW-4343 - Add CategoryPurposeCode

### PR DESCRIPTION
**Pull request by charlieGreenyOB**
added to:
- OBRisk1 (Payments)
- OBReadTransaction6/Data/Transaction

### CODE REVIEW

File: docs/v4.0-draft1/resources-and-data-models/aisp/Transactions.md
Comment (OBPeteS) [inline docs/v4.0-draft1/resources-and-data-models/aisp/Transactions.md:1]: Transaction schema doesn't have a Risk block, the CategoryPurposeCode should be in-line with the other values of the schema

File: docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md
Comment (OBPeteS) [inline docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: CategoryPurposeCode  is missing from UML, will the addressed in a different PR?

**Review by OBPeteS — CHANGES_REQUESTED** [review]

File: docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md
Comment (charlieGreenyOB) [reply docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: From what I understand a new tag/version for `v4.0-draft1` has to be made [here](https://github.com/OpenBankingUK/read-write-api-specs). Then the scripts in this repo will pull them down/generate them accordingly 

@wkrop correct me if I am wrong 

I was just about to do this on the other repo btw

Comment (charlieGreenyOB) [reply docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: see this [commit](https://github.com/OpenBankingUK/read-write-api-docs-pub/commit/6dcbf068ac920c47bad07a2813f397b7feb05979) for referrence

Comment (wkrop) [reply docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: Please try to make a local change  to refer to  the branch/local file. Then run the script again. Please do not push the script change within this PR

Comment (wkrop) [reply docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: @OBPeteS @charlieGreenyOB 

UML files need to be moved manually.
I would suggest to do it within other PR will diagrams.

Comment (OBPeteS) [reply docs/v4.0-draft1/resources-and-data-models/vrp/domestic-vrp-consents.md:1]: Happy for the diagrams to be generated later after swagger changes are done. Please ensure that diagram updates are not missed if skipped in a PR

File: docs/v4.0-draft1/resources-and-data-models/aisp/Transactions.md
Comment (OBPeteS) [reply docs/v4.0-draft1/resources-and-data-models/aisp/Transactions.md:1]: This has been updated

**Review by OBPeteS — APPROVED** [review]

**Review by wkrop — APPROVED** [review]

**Pull request merged**